loadtest/control/simulcontroller/utils.go

```diff
@@ -47,11 +49,25 @@ func genMessage(isReply bool) string {
 	return message
 }
 
+func splitName(name string) (string, string) {
+	reSplit := regexp.MustCompile(`\d+$`)
```

Comment:
The username format isn't configurable. It's set in the `userentity.Config` directly. I've made the assumption that it will be in this format. The tests show how this function behaves when it isn't in this format. 
Do we want to support the case where there is a constant suffix.

Reply:
Yeah, there is one more place where "testuser" is used, during the creation of team admins. But it's all in the main package.

I would say it might be simpler and safer if we just check for the "testuser-" prefix and if it exists, then strip it off and use the remaining part. Otherwise use the original string. The assumption is that if the DB has artificially generated data, it is assumed that it has been generated by our load tests and therefore has the "testuser-" prefix. And if it doesn't have such a prefix, then we assume that it has real world data, for example a customer load testing with their existing DB.

Checking for only the numeric part will break in scenarios where usernames in real world data ends with numbers.